package.json

```diff
@@ -4,8 +4,7 @@
   "description": "Run your tape tests on BrowserStack",
   "main": "index.js",
   "scripts": {
-    "start": "browserify index.js example/index.js -o example/bundle.js && browserstack-runner",
-    "local": "browserify index.js example/index.js -o example/bundle.js && node example/bundle.js",
+    "start": "browserify index.js example/index.js -o example/bundle.js && browserstack-runner | tap-difflet",
```

Comment:
`start` implies a real application-type script for me - I'd be fine with replacing the `test` script with this same thing, since it really is a development test.  I'd be down with renaming the `example` folder to `test`, too.  (Thanks for writing an actual test by the way!)